gitops-generator/pkg/gitops/generate.go

```diff
@@ -40,11 +39,11 @@ func GenerateTektonBuild(outputPath string, component appstudiov1alpha1.Componen
 		tektonResourcesDirName := ".tekton"
 
 		if err := GenerateBuild(appFs, filepath.Join(componentPath, tektonResourcesDirName), component, gitopsConfig); err != nil {
-			return util.SanitizeErrorMessage(fmt.Errorf("failed to generate tekton build in %q for component %q: %s", componentPath, componentName, err))
+			return fmt.Errorf("failed to generate tekton build in %q for component %q: %s", componentPath, componentName, err)
```

Comment:
how come this is removed

Reply:
That is... a good question. I think it's a holdover from earlier testing. This should be reverted